### PR TITLE
Integrate ocpbugs with Bugs row in release_page

### PR DESCRIPTION
**Pull request by MartinPl0**
Added ocpbugs to release_page which previously only showed Bugzilla bugs.

Preserved the existing implementation for status and count display.

### CODE REVIEW

File: lib/errata/errata_requests.py [hunk 1]
```diff
@@ -166,7 +178,26 @@ def format_advisory_data(advisory_data):
...
+                "is_private": jira_issue.get("is_private"),
+                "labels": jira_issue.get("labels"),
+            }
+            jira_issues_details.append(jira_issue_detail)
+            total_jira_issues += 1
+
+            # Counting the JIRA issue statuses
+            if jira_issue_detail["status"] not in jira_issue_summary:
+                jira_issue_summary[jira_issue_detail["status"]] = 0
+            jira_issue_summary[jira_issue_detail["status"]] += 1
```
Comment (ashwindasr) [inline lib/errata/errata_requests.py:200]: Can you explain what you're trying to achieve here? I'm not exactly clear what the difference is between `jira_issue_summary` and `jira_issue_detail`

File: lib/errata/errata_requests.py [hunk 2]
```diff
@@ -189,10 +220,33 @@ def format_advisory_data(advisory_data):
...
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": jira_issue_summary[key],
+                "percent": 0
+            })
+        else:
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": jira_issue_summary[key],
+                "percent": round((jira_issue_summary[key] / total_jira_issues) * 100, 2)
```
Comment (ashwindasr) [inline lib/errata/errata_requests.py:234]: Can I know why we are sending back the percentage? I'm not sure if I see them on the front end, unless I'm missing something? Similarly in Bugzilla as well

Cause we just need the count right?

File: lib/errata/errata_requests.py [hunk 3]
```diff
@@ -189,10 +220,33 @@ def format_advisory_data(advisory_data):
...
+                "bug_status": key,
+                "count": bug_summary[key],
+                "percent": 0
+            })
+        else:
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": bug_summary[key],
+                "percent": round((bug_summary[key] / total_bugs) * 100, 2)
+            })
```
Comment (ashwindasr) [inline lib/errata/errata_requests.py:249]: Also now that both JIRA and bugzilla stores bugs, it might be a good idea to rename the variables and say `jira_bugs` and `bugzilla_bugs` respectively. This can also be a refinement PR if you prefer

**Review by ashwindasr — CHANGES_REQUESTED** [review]

**Comment by ashwindasr** [comment]
Also looks like there are a couple of tox errors. Can you take a look at that as well?

File: lib/errata/errata_requests.py (hunk 1 above)
Comment (MartinPl0) [reply lib/errata/errata_requests.py:200]: **jira_issue_detail["status"]:**
This captures the status of the current JIRA issue we're iterating over.

**if jira_issue_detail["status"] not in jira_issue_summary::**
This checks if the status of the current JIRA issue exists in our jira_issue_summary dictionary. If it doesn't, we initialize a counter for this status in the next line.

**jira_issue_summary[jira_issue_detail["status"]] = 0:**
We're initializing the count of this particular status to zero. This is a setup for our tally of different JIRA issue statuses.

**jira_issue_summary[jira_issue_detail["status"]] += 1:**
This line increments the count of the current status by one, effectively counting the number of occurrences of each status.

**To differentiate between jira_issue_detail and jira_issue_summary:**
jira_issue_detail: Contains specific details about a single JIRA issue, including its status.
jira_issue_summary: Is a dictionary that acts as a counter, keeping track of the number of JIRA issues that have each specific status.

So in short, I'm tallying the number of JIRA issues per status in jira_issue_summary using details from each jira_issue_detail.

File: lib/errata/errata_requests.py (hunk 2 above)
Comment (MartinPl0) [reply lib/errata/errata_requests.py:234]: ah this one should not be there, i had some issues reaching the API so I was trying to send all paramaters in documentations for testing, I will remove it.

File: lib/errata/errata_requests.py (hunk 3 above)
Comment (MartinPl0) [reply lib/errata/errata_requests.py:249]: good idea! Will do it and amend it in this PR

**Comment by ashwindasr** [comment]
Can you deploy it your workspace as well and share the link?

**Comment by ashwindasr** [comment]
And make sure that the modified changes works with the front end as well

**Comment by MartinPl0** [comment]
They work with dev env and hackspace, here is hackspace link:
https://art-dash-hackspace-martin.apps.artc2023.pc3z.p1.openshiftapps.com/dashboard/release/openshift-4.13

**Review by ashwindasr — APPROVED** [review]
